docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -1 +1,162 @@
+
+== Setup
+First we need a local blockchain to develop on. We use `ganache-cli`, so go ahead and install (if you haven't already) and then run a local blockchain using `ganache-cli`
+
+[source,console]
+----
+$ npm install -g ganache-cli
+$ ganache-cli --deterministic
+----
+
+Next we need a project directory to develop our project in. In a separate terminal create the project directory.
+
+[source,console]
+----
+$ mkdir myproject && cd myproject
+----
+
+Unpack the Tutorial Starter Kit. (This could take a little while so maybe make a cup of tea)
+
+[source,console]
+----
+$ npx @openzeppelin/cli unpack tutorial
+----
+
+Initialize the project with OpenZeppelin.
+
+[source,console]
+----
+$ npx openzeppelin init
+? Welcome to the OpenZeppelin SDK! Choose a name for your project myproject
+? Initial project version 1.0.2
+Project initialized. Write a new contract in the contracts folder and run 'openzeppelin create' to deploy it.
+----
+
+Enable the Solidity Hot Loader in `client/config/webpack.js` by setting `disabled: false` in your favorite editor. We can disable this when we need.
+
+[source,js]
+----
+const solidityLoaderOptions = {
+  network: 'development',
+  // you can stop loader from automatic compile/push/upgrade
+  // action by setting disabled flag to true, but it will still
+  // serve .json files from file system
+  disabled: false,
+};
+
+module.exports = {
+  solidityLoader: {
+    test: /\.sol$/,
+    use: [
+      { loader: 'json-loader' },
+      {
+        loader: '@openzeppelin/solidity-loader',
+        options: solidityLoaderOptions,
+      },
+    ],
+  },
+  solidityLoaderOptions,
+};
+----
+
+== Counter contract
+Compile the Counter contract with OpenZeppelin CLI.
+
+[source,console]
+----
+$ npx oz compile
+✓ Compiled contracts with solc 0.5.13 (commit.5b0b510c)
+----
+
+Deploy the Counter contract to our local blockchain.
+
+Run `npx oz create` and use the interactive commands to deploy the Counter contract to the development network and initialize to a value e.g. `23`.
+
+[source,console]
+----
+$ npx oz create
+Nothing to compile, all contracts are up to date.
+? Pick a contract to instantiate Counter
+? Pick a network development
+✓ Added contract Counter
+✓ Contract Counter deployed
+All contracts have been deployed
+? Call a function to initialize the instance after creating it? Yes
+? Select which function * initialize(num: uint256)
+? num (uint256): 23
+✓ Setting everything up to create contract instances
+✓ Instance created at 0xCfEB869F69431e42cdB54A4F4f105C19C080A601
+0xCfEB869F69431e42cdB54A4F4f105C19C080A601
+----
+
+== App
+We will now start the app.
+
+[source,console]
+----
+$ cd client
+$ npm run start
+----
+
+The app will open in a browser on http://localhost:3000/
```

Comment:
Consider merging these sentences into a single paragraph so that the text isn't too broken up

Reply:
Updated (slightly)